web/pages/links.tsx

```diff
@@ -71,6 +68,35 @@ export default function LinkPage() {
           don&apos;t yet have a Manifold account.
         </p>
         <Subtitle text="Your Manalinks" />
+        <ManalinksDisplay
+          unclaimedLinks={unclaimedLinks}
+          highlightedSlug={highlightedSlug}
+        />
+      </Col>
+    </Page>
+  )
+}
+
+function ManalinksDisplay(props: {
+  unclaimedLinks: Manalink[]
+  highlightedSlug: string
+}) {
+  const { unclaimedLinks, highlightedSlug } = props
+  const [page, setPage] = useState(0)
+  const start = page * LINKS_PER_PAGE
+  const end = start + LINKS_PER_PAGE
+  const displayedLinks = unclaimedLinks.slice(start, end)
+
+  if (unclaimedLinks.length === 0) {
+    return (
+      <p className="text-gray-500">
+        You don't have any unclaimed manalinks. Send some more to spread the
+        wealth!
+      </p>
+    )
+  } else {
+    return (
+      <>
         <Col className="grid w-full gap-4 md:grid-cols-2">
           {displayedLinks.map((link) => {
```

Comment:
I think you need a `key` prop in the `<ManalinkCardFromView>` component below. 

https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
You could also use the short form of an anonymous function without a return statement: 
`displayedLinks.map((link) => <ManalinkCardFromView ... />)